types/report.go

```diff
@@ -1,14 +1,28 @@
 package types
 
+import (
+	"fmt"
+	"log"
+	"regexp"
+	"strings"
+
+	"github.com/ms-henglu/armstrong/coverage"
+)
+
 type PassReport struct {
 	Resources []Resource
 }
 
 type Resource struct {
+	ApiPath string
 	Type    string
 	Address string
 }
 
+type CoverageReport struct {
```

Comment:
>CoverageReport [](http://example.com/codeflow?start=5&length=14)

could it be moved into `coverage` package?

Reply:
I think it's better to keep it for other `*Report` struct is in here.